tutor/src/components/task-plan/select-topics.cjsx

```diff
@@ -28,8 +30,7 @@ SelectTopics = React.createClass
     @props.onSectionChange?(sectionIds)
 
   renderDialog: ->
-    {courseId, planId, selected, hide, header, primary, cancel} = @props
-    ecosystemId = CourseStore.get(courseId).ecosystem_id
+    {courseId, planId, selected, hide, header, primary, cancel, ecosystemId} = @props
 
```

Comment:
This is the bug fix.  We're now reading the ecosystem from @props, rather than the course.

@Dantemss note that I didn’t default the ecosystem to the course.  I now think that’s a bad idea all around, since we're using the ecosystem is required on @props, and that's also what we're using to load the TocStore below on line 58.